Formula/sqlc.rb

```diff
@@ -0,0 +1,43 @@
+class Sqlc < Formula
+  desc "Generate type safe Go from SQL"
+  homepage "https://sqlc.dev/"
+  url "https://github.com/kyleconroy/sqlc/archive/v1.6.0.tar.gz"
+  sha256 "a95a123f29a71f5a2eea0811e4590d59cbc92eccd407abe93c110c738fc4740b"
+  license "MIT"
+  head "https://github.com/kyleconroy/sqlc.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args, "./cmd/sqlc"
```

Comment:
```suggestion
    system "go", "build", *std_go_args, "-ldflags", "-s -w", "./cmd/sqlc"
```

Reply:
Can you explain why these flags should be added?

Reply:
It produces binaries of smaller sile (https://blog.filippo.io/shrink-your-go-binaries-with-this-one-weird-trick/):
``` 
# go build -trimpath -o /usr/local/Cellar/sqlc/1.6.0/bin/sqlc ./cmd/sqlc
$ du -h /usr/local/Cellar/sqlc/1.6.0/bin/sqlc 
 37M	/usr/local/Cellar/sqlc/1.6.0/bin/sqlc
```
```
# go build -trimpath -o /usr/local/Cellar/sqlc/1.6.0/bin/sqlc -ldflags -s -w ./cmd/sqlc
$ du -h /usr/local/Cellar/sqlc/1.6.0/bin/sqlc 
 28M	/usr/local/Cellar/sqlc/1.6.0/bin/sqlc
```

And most of `go`-formulae in homebrew-core are using these ldflags.

Reply:
Isn't that part of `std_go_args`?

Reply:
[Nope](https://github.com/Homebrew/brew/blob/2.7.0/Library/Homebrew/formula.rb#L1434-L1437):

```ruby
  # Standard parameters for Go builds.
  def std_go_args
    ["-trimpath", "-o", bin/name]
  end
```

Reply:
I wonder if we should move `ldflags -s -w` into std_go_args.